ci/deploy.sh

```diff
@@ -5,7 +5,7 @@
 
 # Assumes 'gcloud auth' has been done with an appropriate deploy account.
 
-# This script is executed by the circle.yml config when the conditions
+# This script is executed by the old_circle.yml config when the conditions
```

Comment:
I believe we're still using this. We shouldn't change this reference?

Reply:
changed to new config.yml